content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -1,296 +1,425 @@
+Now you're ready to run your project's Game Launcher.
 
-1. Open a command line window and change to your project's release build install directory.
+Run `GameLauncher.exe` from your project game release layout, which is located in the `<install>/bin/Windows/release/<build>` directory. Examples of the path are
 
-    - For monolithic release builds:
-        ```cmd
-        cd C:/o3de-projects/MyProject/install/bin/Windows/release/Monolithic
-        ```
+- `C:/MyProject/install/bin/Windows/release/Default` -- for non-monolithic.
 
-    - For non-monolithic release builds:
-        ```cmd
-        cd C:/o3de-projects/MyProject/install/bin/Windows/release/Default
-        ```
+- `C:/MyProject/install/bin/Windows/release/Monolithic` -- for monolithic.
 
-2. Run the Game Launcher. This opens the Game Launcher and loads the starting level.
-    
-    ```cmd
-    ./MyProject.GameLauncher.exe
-    ```  
+Now you can distribute your project to other Windows devices. To run the Game Launcher, you must install [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160).
+
+
+## Debugging
+
+If you encounter issues when you build a project for release, try the following techniques to help you debug.
+
+### Compile with optimizations disabled and debug symbols enabled
+
+In Visual Studio 2019, you can compile with optimizations disabled and debug symbols enabled. This allows Visual Studio 2019's debugging tools to produce more helpful information that can help you debug.
+
+To do this, make the following modifications to `Configurations_msvc.cmake` in the `<engine>/cmake/Platform/Common/MSVC` directory. In `ly_append_configurations_options`, under `COMPILATION _RELEASE`:
+ - Change `/Od` to `/Ox`.
+ - Add `/Zi`.
+
+After making those modifications, `ly_append_configurations_options` should look like this:
+
+```
+ly_append_configurations_options(
+
+    # ...
+
+    COMPILATION_RELEASE
+        /Od             # Enable debug symbols
+        /Ob2            # Inline any suitable function
+        /Ot             # Favor fast code over small code
+        /Oi             # Use Intrinsic Functions
+        /Oy             # Omit the frame pointer
+        /Zi             # Generate debugging information (no Edit/Continue)
     
-Now you can distribute your project to other Windows devices. To run the Game Launcher, [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160) must be installed.
\ No newline at end of file
+    # ... 
+```
+
+### Create a `profile` build
+
+You can build with the `profile` configuration to debug a non-monolithic or monolithic build and its `.pak` files. The `profile` configuration creates log files that provide information to help you debug. For information on O3DE log files, refer to [Log Files](\docs\user-guide\appendix\log-files.md).
```

Comment:
```suggestion
To debug a non-monolithic or monolithic build and its `.pak` files, you can build with the `profile` configuration. The `profile` configuration creates log files that provide information to help you debug. For information on O3DE log files, refer to [Open 3D Engine Log Files](/docs/user-guide/appendix/log-files).
```
* Re-ordered the first sentence to lead with the intent/infinitive phrase ("To debug...").
* Fixed the broken link path and updated the name of the destination topic for consistency.